.github/workflows/qc.yml

```diff
@@ -51,3 +50,12 @@ jobs:
 
       - name: Test with pytest and generate coverage file
         run: poetry run tox -e py
+
+      #-----------------------------------------------------
+      # QC with pandas 1.x.x
+      # Note: The lock file will always have pandas > 2.0.0
+      #-----------------------------------------------------
+      - name: Test with pandas<2.0.0
+        run: |
+          poetry add "pandas<2.0.0"
+          poetry run pytest
```

Comment:
Do replicate the condiations above exactly, wouldnt this test have to run is tox?

Reply:
Shouldn't matter, it is exactly the same. But just for the sake of clarity, I changed it.